tensorflow/compiler/tf2tensorrt/utils/trt_lru_cache.h

```diff
@@ -146,10 +146,19 @@ struct EngineContext {
   }
 
   // In explicit batch mode, we maintain a vector of contexts for each engine,
-  // where each context is created for a different profile. The
+  // where each context is created for a different profile. The are two reasons
+  // for this:
+  // - In TRT 6 it is not possible to switch profile after it was set
```

Comment:
s/profile after it was/profiles after they are/  or  s/profile after it was/a profile after it is/

Reply:
Fixed.